custom_components/solcast_solar/solcastapi.py

```diff
@@ -2133,9 +2144,8 @@ async def __migrate_undampened_history(self):
                     _LOGGER.info("Dampening forecasts for today onwards for site %s", site)
                 else:
                     continue
-                day_start = self.get_day_start_utc()
                 for interval, forecast in forecasts[site].items():
-                    if interval >= day_start:
+                    if interval >= self.get_day_start_utc():
```

Comment:
OK, so as I step through and review the changes, this is the first one that stands out and looks odd.

The others all make sense: I can see how you're changing the call to get_day_start_utc to accept a parameter for the number of days into the future or the past, and all the changes look OK.

But this test swaps from _is the interval >= to the_ `day_start` to _is the interval >=_ **day_start_UTC** which is a different thing, is it not?

If not, cool, but better to ask and be wrong than remain silent.

Reply:
`day_start = self.get_day_start_utc()` from two lines up is now gone, cutting out the middle-man variable that was not used elsewhere in the method.

But your peepers are clearly working 👀.

Reply:
Well, not that well, but looking for oddness like varname=int instead of varname=init.

Did pick out some new frames today though for when I get new specs. I have a big head. Frames are hard to find.